scaffold/cmd/templates/source/.github/workflows/test.yaml.tpl

```diff
@@ -22,7 +22,10 @@ jobs:
       - name: golangci-lint
         uses: golangci/golangci-lint-action@v3
         with:
-          version: v1.51.2
+          version: v1.55.2
+          skip-pkg-cache: true
+          skip-build-cache: true
```

Comment:
We use this approach in our tests to only keep the lint results cache. `setup-go` handles build and package cache already